### PR TITLE
Feat: #116 Add encryption type dropdown

### PR DESCRIPTION
**Pull request by hoailinh1210**
### Summary (Please recap what you changed or fixed)

Import Keyphrase
image

Recovery Phrase:
image

### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [x] Provide the screenshots due to UI changed or bug fixed
-   [x] My code has covered these test cases (please list down your tested cases):

#### Wallet with encryption type ED25519
1. Create a wallet with an encryption type ED25519
2. Get Keyphrase and public address (*)
3. Import keyphrase from (2) -> Get public address (**)

Actual: Same public address between (*) and (**)

#### Wallet with encryption type Secp256k1
1. Create a wallet with an encryption type Secp256k1
2. Get Keyphrase and public address (*)
3. Import keyphrase from (2) -> Get public address (**)

Actual:  Same public address between (*) and (**)

### CODE REVIEW

File: client/webpack/extensionConfig.js [hunk 1]
```diff
@@ -62,15 +62,15 @@ module.exports = (dir) =>
...
+		// 	splitChunks: {
+		// 		cacheGroups: {
+		// 			vendors: {
+		// 				test: /[\\/]node_modules[\\/]/,
+		// 				name: 'vendors',
+		// 				chunks: 'all',
+		// 			},
+		// 		},
+		// 	},
+		// },
```
Comment (duc-q) [inline client/webpack/extensionConfig.js:75]: Somehow this option prevents connection from the extension to service worker so we had to comment it out

**Comment by duc-q** [comment]
@hoailinh1210 

Please help to pull latest code, I fixed up some elements jumping in Import Wallet screen.

Thanks,

File: client/src/components/web-extension/Common/SelectEncryptionType/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                        <span>
+                            <InfoIcon />
+                        </span>
+                    </OverlayTrigger>
+                </span>
+
+            </div>
+                <Select
+	className="cd_we_dropdown"
+	placeholder="Select the algo"
```
[the remark this answers is not in the document]
Comment (gaupoit) [reply client/src/components/web-extension/Common/SelectEncryptionType/index.js:53]: Please refer to mobile version.

[the remark this answers is not in the document]
Comment (hoailinh1210) [reply client/src/components/web-extension/Common/SelectEncryptionType/index.js]: Removed this, I am setting the default option is Ed25519

File: client/webpack/extensionConfig.js (hunk 1 above)
Comment (djhahe) [reply client/webpack/extensionConfig.js:75]: I'll check it and update later

**Review by djhahe — APPROVED** [review]

**Comment by djhahe** [comment]
@hoailinh1210 Please help to resolve conflict and merge

**Comment by hoailinh1210** [comment]
> @hoailinh1210 Please help to resolve conflict and merge

Fixed this conflict 👍

**Pull request merged**